doc/whats_new.rst

```diff
@@ -172,6 +172,10 @@ Enhancements
      :issue:`7674` by:user:`Yichuan Liu <yl565>`.
 
 Bug fixes
+
+   - Fixed a bug where :func:`sklearn.grid_search.BaseSearchCV.inverse_transform`
```

Comment:
You should not add it between `Bug fixes` and `.........`.

By the way, you could probably merge this entry with the one of `Akshay0724` 4 entries below, which was exactly the same on a different file.